script/Arcueid/Day 5/05_00_ARC05_3_2B.txt

```diff
@@ -186,21 +187,22 @@ I won't be able to defeat him with my strength tonight, but I can at least put o
 [sha:848e9ebf26d618c10c4d76c015464b74b56f58ec]{
 -- Page 260, Offset 9238.
 -- 直接組み伏せるとか無駄すぎるし。今まで一度もしたコトないし。する必要性が感じられないし」
-It would be futile to hold him down directly. Besides, I've never done it before, and I don't feel like starting now."
+It would be futile to hold him down directly. Besides, I've never done it before, nor do I feel the need to do it."
```

Comment:
Kinda like the idiomatic ver, though it is maybe a liberty. If we keep the new ver, I'd drop the `do it` on the end.

Reply:
_I feel the need for speed_